meson.build

```diff
@@ -35,7 +35,9 @@ else
   error('Cannot found header mustache.hpp')
 endif
 
-libzim_dep = dependency('libzim', version : '>=9.0.0', static:static_deps)
+libzim_dep = dependency('libzim', version:'>=9.0.0', static:static_deps)
+libzim_dep = dependency('libzim', version:'<10.0.0', static:static_deps)
```

Comment:
IMHO, this doesn't add a version requirement. It rather overrides the preceding one (so only `version:'<10.0.0'` stays in effect). Yes, the build will likely fail if `libzim` newer than `9.0.0` is not found, but then meson is allowed to pick up a version older than 10.0.0 (including a version lower than 9.0.0).

Reply:
The correct way is to [specify multiple restrictions](https://mesonbuild.com/Reference-manual_functions.html#dependency_version):  `version:['>=9.0.0', '<10.0.0']`

I am going to fix it in #1133

Reply:
Hmmm, this approach has been validated on an other repo. by @mgautierfr. I don't remember the results of testing I made at that time... but AFAIK it tests the first one and tests then the second one... which is what we want.

@veloman-yunkan Do you have a better formalism?

Reply:
Done